cassandra-3.11/Dockerfile

```
@@ -1,13 +1,17 @@
 FROM openjdk:8u252-slim
 
-ARG STARGATE_VERSION=v1.0.0
+ARG STARGATE_VERSION=v1.0.1
 
-RUN apt update && apt install curl unzip iproute2 libaio1 -y
-RUN curl -L https://github.com/stargate/stargate/releases/download/${STARGATE_VERSION}/stargate-jars.zip > stargate-jars.zip
+RUN apt update -qq \
+    && apt install curl unzip iproute2 libaio1 -y \
+    && apt autoremove --yes \
+    && apt clean all \
+    && rm -rf /var/lib/{apt,dpkg,cache,log}/
```

I've seen issues with this line before, upstream images indicate this is safe

```suggestion
    && rm -rf /var/lib/apt/lists/*
```